### DISCUSSION TITLE
Bug in Grafana

### DISCUSSION THREAD

**Discussion by pengnist** [srsran/srsRAN_Project#582] [General Help]
Hi Community,

I would like to report a bug in Grafana webUI: 

When I connected three srsUEs to the gnb, the Num Cells always shows No Data and Active UEs is always 1. 
![image](https://github.com/srsran/srsRAN_Project/assets/71417513/edbe2d8c-8d24-486d-860b-71a32e4ffef2)

When checking the Query on Grafana, Num Cells's query shows an error "invalid: no column "pci" exists". I suspect the Query for "Active UEs" was supposed to be for "Num Cells". When pasting the old Query for "Active UEs" in the previous release, it still works for this release.

**Comment by pgawlowicz** [comment]
@pengnist did it improve with the recent code release?

**Reply by pengnist** [reply]
I am sorry I was on vacation in the past weeks. Will test and get back to you soon.
